packages/appeals-service-api/__tests__/developer/appeals-case-data.test.js

```diff
@@ -58,12 +58,13 @@ describe('appeals-case-data', () => {
 	it('should return valid appeals based on LPA code and status', async () => {
```

Comment:
probably worth either mentioning that this also tests for order or creating a separate test so it's more explicit like the unit test has